src/Widget/Block.php

```diff
@@ -33,8 +35,8 @@
  * Second parameter defines if block content should be outputted which is desired when rendering its content but isn't
  * desired when redefining it in subviews.
  *
- * @method static Block begin()
- * @method static Block end()
+ * @method static Block begin(WebView $view)
+ * @method static Block end(WebView $view)
```

Comment:
All these annotations could be removed, if you use `@return static` i `Widget` for related methods.

Reply:
I'n not sure about it in case of different IDEs. For example, my version of PhpStorm get it right without phpdocs at all (using just return type hint `self`, as far as I know).

Reply:
It is already convention used in Yii 2 and it seems that it does not create any problems.